deploy/operator.yaml

```diff
@@ -53,7 +53,7 @@ spec:
       initContainers:
         - name: non-root-enabler
           image: busybox
-          # Creates folder /var/lib/kubelet/seccomp/operator and sets 2000:2000 as its owner.
+          # Creates folder /var/lib/seccomp-operator, sets 2000:2000 as its owner and symlink it to /var/lib/kubelet/seccomp/operator.
           # This is required to allow the main container to run as non-root.
           command: ["/bin/sh", "-c"]
           args:
```

Comment:
Oops, the linking seems missing now :see_no_evil: 
/hold

Reply:
Yep, it was an issue on conflict resolution.. PTAL